.github/workflows/wf-build-release-ci.yml

```diff
@@ -10,11 +10,11 @@ jobs:
     env:
         ACTIONS_ALLOW_UNSECURE_COMMANDS: true
     steps:
-    - uses: actions/checkout@v1
+    - uses: actions/checkout@v4
       name: Checkout Code
     
     - name: Install additional .NET SDKs
-      uses: actions/setup-dotnet@v1
+      uses: actions/setup-dotnet@v4
```

Comment:
Just FYI, I've had numerous problems with other GitHub workflows that are still using v1 of `actions/setup-dotnet`.

Reply:
If it works, then let's use v4. I just ised v1 because it was the up-to-date version when setting up the workflow.